serve-authenticated.py

```diff
@@ -0,0 +1,21 @@
+import http.server
+import socketserver
+
+class AuthenticatedHTTPRequestHandler(http.server.SimpleHTTPRequestHandler):
+  def do_GET(self):
+    #print(self.request)
+    if self.headers['Authorization']:
+        if self.headers['Authorization'].startswith('Basic '):
+            if self.headers['Authorization'][6:] == "dGVzdDp0ZXN0":
```

Comment:
is `dGVzdDp0ZXN0` the (hashed) password? 

It is not very clear to me how this Python script should be used in testing.

Reply:
Yes, these are the Basic Authentication base64-encoded username and password - I changed the code a bit to make this easier to see, and added a `CONTRIBUTING.md` describing that you need to have this server running for the `sbtix/private-auth` test.